docs/get-started-dapr-secret-store-configuration-provider-aspnet-core.md

```diff
@@ -0,0 +1,352 @@
+# Dapr Secret Store Configuration Provider in ASP.NET Core
```

Comment:
This doc is not needed here, I would just add a readme.md with the sample itself. There is no need to add code snippets in the doc as it becomes hard to manage code in docs when the apis change, users can look at the samples to understand it. (I will also remove the code from docs for actors.)

Reply:
@amanbha just moved the important commands to README.md.